package.json

```diff
@@ -40,10 +40,12 @@
         "@pika/plugin-bundle-web": "^0.5.0",
         "@pika/plugin-copy-assets": "^0.5.0",
         "@pika/plugin-standard-pkg": "^0.5.0",
+        "@rdkit/rdkit": "^2022.3.2",
```

Comment:
The new setup is using the npm release of the JavaScript rdkit MinimalLib https://www.npmjs.com/package/@rdkit/rdkit